src/main/java/picard/sam/SamErrorMetric/ReadBaseStratification.java

```diff
@@ -640,6 +829,30 @@ public String getSuffix() {
      */
     public static final NsInReadStratifier nsInReadStratifier = new NsInReadStratifier();
 
+    /** Stratify by Insertions in the read cigars.  */
+    public static final CigarOperatorsInReadStratifier insertionsInReadStratifier = new CigarOperatorsInReadStratifier(CigarOperator.I);
+
+    /** Stratify by Deletions in the read cigars.  */
+    public static final CigarOperatorsInReadStratifier deletionsInReadStratifier = new CigarOperatorsInReadStratifier(CigarOperator.D);
+
+    /**  Stratify by Indels in the read cigars.  */
+    public static final IndelsInReadStratifier indelsInReadStratifier = new IndelsInReadStratifier();
+
+    /**
+     * Stratifies into the number of bases in an insertion
+     */
+    public static final IndelLengthStratifier indelLengthStratifier = new IndelLengthStratifier();
+
+    /**
+     * Stratifies by whether or not the current RecordAndOffset is an insertion
+     */
+    public static final IsInsertionStratifier isInsertionStratifier= new IsInsertionStratifier();
```

Comment:
You're right, the use of this isn't really apparent when the user is only interested in the metrics. With the filtering in mind though, it would be handy to be able to find all insertions of size 2, for example, which would require a criterion such as `indel_length == 2 && is_insertion == true`. I'll leave this open until we have determined if the filtering should be done in a separate tool or not

Reply:
I have removed the IsInsertion and IsDeletion stratifiers as they are no longer needed